### ISSUE TITLE
cookbooks missing from readthedocs

### ISSUE THREAD

**Issue by rainwoodman** [bccp/nbodykit#499]
Cookbooks are missing from readthedocs. When I build the docs locally they also don't show up. I see warnings like this:

```
File.dtype
/home/yfeng1/source/nbodykit/docs/source/catalogs/mock-data.ipynb:273: WARNING: undefined label: cookbook/lognormal-mocks.ipynb (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/catalogs/mock-data.ipynb:641: WARNING: undefined label: cookbook/hod-mocks.ipynb (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/catalogs/on-demand-io.ipynb:32: WARNING: unknown document: dask:array-overview
/home/yfeng1/source/nbodykit/docs/source/cookbook/index.rst:39: WARNING: undefined label: cookbook/lognormal-mocks.ipynb (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/cookbook/index.rst:42: WARNING: undefined label: cookbook/hod-mocks.ipynb (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/cookbook/index.rst:52: WARNING: undefined label: cookbook/painting.ipynb#painting-the-overdensity-field (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/cookbook/index.rst:55: WARNING: undefined label: cookbook/painting.ipynb#painting-the-line-of-sight-momentum-field (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/cookbook/index.rst:58: WARNING: undefined label: cookbook/painting.ipynb#painting-multiple-species-of-particles (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/cookbook/index.rst:65: WARNING: undefined label: cookbook/fftpower.ipynb (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/cookbook/index.rst:69: WARNING: undefined label: cookbook/interlacing.ipynb (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/cookbook/index.rst:73: WARNING: undefined label: cookbook/interpolation-windows.ipynb (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/cookbook/index.rst:77: WARNING: undefined label: cookbook/convpower.ipynb (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/cookbook/index.rst:81: WARNING: undefined label: cookbook/boss-dr12-data.ipynb (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/cookbook/index.rst:84: WARNING: undefined label: cookbook/angular-paircount.ipynb (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/mesh/creating.ipynb:117: WARNING: undefined label: cookbook/interpolation-windows.ipynb (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/mesh/creating.ipynb:138: WARNING: undefined label: cookbook/interlacing.ipynb (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/mesh/creating.ipynb:187: WARNING: undefined label: cookbook/painting.ipynb#painting-multiple-species-of-particles (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/mesh/creating.ipynb:198: WARNING: undefined label: cookbook/painting.ipynb#painting-the-line-of-sight-momentum-field (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/mesh/painting.rst:70: WARNING: undefined label: cookbook/painting.ipynb#painting-the-line-of-sight-momentum-field (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/mesh/painting.rst:76: WARNING: undefined label: cookbook/painting.ipynb#painting-multiple-species-of-particles (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/results/algorithms/fftpower.ipynb:24: WARNING: undefined label: cookbook/fftpower.ipynb (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/results/algorithms/fftpower.ipynb:427: WARNING: undefined label: cookbook/painting.ipynb (if the link has no caption the label must precede a section header)
/home/yfeng1/source/nbodykit/docs/source/results/algorithms/projected-fftpower.rst:5: WARNING: more than one target found for cross-reference 'ProjectedFFTPower': nbodykit.algorithms.ProjectedFFTPower, nbodykit.algorithms.fftpower.ProjectedFFTPower

```

@nickhand could you take a look when you have the chance?

cc: @michaelJwilson

**Comment by rainwoodman** [comment]
Probably because make ipynb fails?

**Comment by nickhand** [comment]
🤔
I'll take a look

**Comment by rainwoodman** [comment]
Fixed in 30bbfa8a819cb. absolute path is needed.
